nixos/modules/system/activation/switch-to-configuration.pl

```diff
@@ -474,6 +476,13 @@ sub filterUnits {
 print STDERR "setting up tmpfiles\n";
 system("@systemd@/bin/systemd-tmpfiles", "--create", "--remove", "--exclude-prefix=/dev") == 0 or $res = 3;
 
+# We need to start nscd before any other service, since they might need
```

Comment:
Not a fan of yet another special case

Reply:
I don't see a way around it without that does not involve getting rid of nscd.